samples/bluetooth/mesh/light_lc/README.rst

```diff
@@ -0,0 +1,189 @@
+.. _bluetooth_mesh_light_lc:
+
+Bluetooth: Mesh Light Control
+#############################
+
+The Bluetooth Mesh Light Control sample demonstrates how to set up a light control Mesh server model application and control a dimmable LED with the Bluetooth Mesh, using the :ref:`bt_mesh_onoff_readme`.
+
+Overview
+********
+
+This sample is split into three source files:
+
+* A :file:`main.c` file to handle initialization.
+* A file for handling Mesh models, :file:`model_handler.c`.
+* A file for handling PWM driven control of the dimmable LED, :file:`lc_pwm_led.c`.
+
+After provisioning and configuring the Mesh models supported by the sample in the `nRF Mesh mobile app`_, you can control the dimmable LED on the development kit from the app.
+
+Provisioning
+============
+
+The provisioning is handled by the :ref:`bt_mesh_dk_prov`.
+
+Models
+======
+
+The following table shows the Mesh light controller composition data for this sample:
+
+.. table::
+   :align: center
+
+   +---------------+-------------------------------+----------------------------+
+   | Element 1     | Element 2                     | Element 3                  |
+   +===============+===============================+============================+
+   | Config Server | Gen. Level Server             | Gen. OnOff Server          |
+   +---------------+-------------------------------+----------------------------+
+   | Health Server | Gen. OnOff Server             | Light Control Server       |
+   +---------------+-------------------------------+----------------------------+
+   |               | Gen. DTT Server               | Light Control Setup Server |
+   +---------------+-------------------------------+----------------------------+
+   |               | Gen. Power OnOff Server       |                            |
+   +---------------+-------------------------------+----------------------------+
+   |               | Gen. Power OnOff Setup Server |                            |
+   +---------------+-------------------------------+----------------------------+
+   |               | Light Lightness Server        |                            |
+   +---------------+-------------------------------+----------------------------+
+   |               | Light Lightness Setup Server  |                            |
+   +---------------+-------------------------------+----------------------------+
```

Comment:
I used a table generator from: https://www.tablesgenerator.com/text_tables.

The reason why i choose to use this formating was because i had a hard time making the manual formating work as i wished, so i gave up and used the generator. I thought that since the rendering was identical it would not matter.